client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -378,15 +380,13 @@ export const SeoForm = React.createClass( {
 			showAdvancedSeo,
 			showWebsiteMeta,
 			site,
+			siteSettings,
 			isFetchingSite,
 			isSeoToolsActive,
 			isVerificationToolsActive,
 			translate,
 		} = this.props;
 		const {
-			settings: {
-				blog_public = 1
-			} = {},
```

Comment:
Pretty fancy destructuring assignment going on here. This is just an aside, but I think anything deeper than one level is confusing for folks to read. So 👍  for removing this.